packages/direflow-component/src/WebComponentFactory.tsx

```diff
@@ -192,10 +195,9 @@ class WebComponentFactory {
       /**
        * Generate react props based on properties and attributes.
        */
-      // eslint-disable-next-line class-methods-use-this
       private reactProps(): any {
-        factory.reflectPropertiesToAttributes();
-        return { ...factory.componentProperties };
+        this.syncronizePropertiesAndAttributes();
+        return { ...this._properties };
```

Comment:
this won't lead to another shallow copy issue, will it?

Reply:
It won't - but can actually be returned directly without the spread.
I'll fix this in the morning 👍